server/org.eclipse.emfcloud.ecore.glsp/src/main/java/org/eclipse/emfcloud/ecore/glsp/EcoreServerLauncher.java

```diff
@@ -39,6 +41,6 @@ private static int getPort(String[] args) {
 				return Integer.parseInt(args[i+1]);
 			}
 		}
-		throw new IllegalArgumentException("No port is defined for ECORE-GLSP. Specify a port as a command line argument with '--port <portnumber>'");
+		return DEFAULT_PORT;
 	}
 }
```

Comment:
Maybe log an info message that no port was defined and the default port is used instead.

Reply:
:+1: